zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/publishing/cacheAPI/NotificationPayloadCacheApi.java

```diff
@@ -0,0 +1,146 @@
+package com.github.onsdigital.zebedee.model.publishing.cacheAPI;
+
+import com.github.onsdigital.zebedee.model.publishing.PublishNotification;
+import com.github.onsdigital.zebedee.util.URIUtils;
+import com.google.gson.annotations.SerializedName;
+import org.apache.commons.lang.StringUtils;
+
+import java.net.MalformedURLException;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+
+public class NotificationPayloadCacheApi {
```

Comment:
The Legacy Cache API does not have notifications, and by the look of this file, this is just the payload that the Legacy Cache API accepts. So a more accurate name would be `LegacyCacheApiPayload`.